2024.5 (#32553)
home-assistant/home-assistant.github.io: source/_posts/2024-05-01-release-20245.markdown

like to see, they may not be in the desired order.

To improve this, [@karwosts] and [@piitaya] have added the ability to reorder
the features of the tile card. For your thermostat, that means you can now
reorder the HVAC modes or presets to your liking.

<p class='img'>
    <img src="/images/blog/2024-05/tile-card-reorder-features.gif" alt="Screenrecording showing how you can now reorder the HVAC modes on the thermostat shown in a tile card.."/>
    You can now reorder the features of the tile card.
</p>

[@karwosts]: https://github.com/karwosts
[@piitaya]: https://github.com/piitaya

## Adjust the device names on the energy dashboard

Another outstanding quality-of-life improvement from [@karwosts] is the ability
to customize the display name for energy devices on the energy dashboard.

The entity's name representing the device you use on the energy dashboard
commonly contains additional or redundant information. With this new feature,
you can customize the display name to make it more meaningful.

For example, if the entity is named _“Attic Boiler Daily Energy Usage”_
and is shown as that today, you can now change the name displayed on the energy
dashboard to just _“Boiler”_.

> <img class="no-shadow" src='/images/blog/2024-05/energy-device-names.png' alt='Screenshots showing you can now assign a different display name to energy sensor of devices that show on your energy dashboard.'>

Reviewer: ```suggestion
<img class="no-shadow" src='/images/blog/2024-05/energy-device-names.png' alt='Screenshots showing you can now assign a different display name to the energy sensor of devices that show on your energy dashboard.'>
```

{% details "Insteon" %}

The Insteon configuration panel, which was previously on the Home Assistant
sidebar, has moved to the integration page in
**Settings -> Device & services -> Insteon -> Configure**.

([@teharris1] - [#105581]) ([documentation](/integrations/insteon))

[@teharris1]: https://github.com/teharris1
[#105581]: https://github.com/home-assistant/core/pull/105581

{% enddetails %}

{% details "Netatmo" %}

The state of the Netatmo wind and gust direction sensor provided by
the weather station now exposes the state in lowercase.

([@joostlek] - [#115413]) ([documentation](/integrations/netatmo))

[@joostlek]: https://github.com/joostlek
[#115413]: https://github.com/home-assistant/core/pull/115413

---

The state of the Netatmo health index sensor provided by the weather station
now exposes the state in lowercase.

([@joostlek] - [#115409]) ([documentation](/integrations/netatmo))

[@joostlek]: https://github.com/joostlek
[#115409]: https://github.com/home-assistant/core/pull/115409

{% enddetails %}

{% details "Recorder" %}

Previously, `recorder.purge_entities` called with no arguments would purge all
entities. This PR makes this call raise an error instead. 

> It was too easy for a user to accidentally purge his entire database using the

Reviewer: ```suggestion
It was too easy for users to accidentally purge their entire database using the
```
